web/src/components/atomic/Icon.vue

```diff
@@ -75,6 +75,7 @@ export type IconNames =
   | 'gitea'
   | 'gitlab'
   | 'bitbucket'
+  | 'bitbucket_dc'
```

Comment:
This is just the type. You also need to add an icon above or at least add it to the exisiting bitbucket icons `v-if`